opentasks/src/main/res/values-da/strings.xml

```diff
@@ -0,0 +1,289 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <!-- Manifest stuff -->
+    <string name="app_name">OpenTasks</string>
+
+    <!-- Task list -->
+    <string name="menu_add_task">Tilføj opgave</string>
+    <string name="menu_add_task_condensed">Tilføj</string>
+    <string name="menu_show_completed">Vis afsluttede opgaver</string>
+    <string name="menu_show_completed_condensed">Vis afsluttede opgaver</string>
+    <string name="menu_sync_now">Opdater</string>
+    <string name="menu_sync_now_condensed">Opdater</string>
+    <string name="menu_search_hint">Søg</string>
+    <string name="toast_task_created">\"%1s\" tilføjet.</string>
+    <string name="toast_task_completed">\"%1s\" afsluttet.</string>
+    <string name="toast_task_uncompleted">\"%1s\" genåbnet.</string>
+    <string name="toast_x_removed">\"%1s\" fjernet.</string>
+    <string name="task_group_title_list">Opgaver</string>
+    <string name="task_group_title_start">Opgaver starter</string>
+    <string name="task_group_title_due">Forfaldne opgaver</string>
+    <string name="task_group_title_priority">Opgaveprioritet</string>
+    <string name="task_group_title_progress">Opgave fremskridt</string>
+    <string name="task_group_title_default">Opgaver</string>
+    <string name="quick_add_task_created">Opgave oprettet</string>
+    <string name="initial_local_task_list_name">Mine opgaver</string>
+
+    <plurals name="number_of_tasks">
+        <item quantity="one">%d opgave</item>
+        <item quantity="other">%d opgaver</item>
+    </plurals>
+
+    <!-- Task detail view -->
+    <string name="activity_task_details_edit">Edit</string>
+    <string name="activity_task_details_edit_task">Rediger opgave</string>
+    <string name="activity_task_details_delete">Slet</string>
+    <string name="activity_task_details_delete_task">Slet opgave</string>
+    <string name="activity_task_details_complete">Komplet</string>
+    <string name="activity_task_details_complete_task">Komplet opgave</string>
+    <string name="activity_task_details_pin">Pin</string>
+    <string name="activity_task_details_pin_task">Pin opgave</string>
+    <string name="opentasks_details_no_task_selected">Ingen opgave valgt/string>
```

Comment:
Some of the strings, like this one are not properly terminated, resulting in an error (see https://travis-ci.org/dmfs/opentasks/builds/596938209#L2690).

Reply:
Fixed the missing <